perfkitbenchmarker/benchmark_spec.py

```diff
@@ -387,8 +393,10 @@ def DeleteVm(self, vm):
 
   def PickleSpec(self):
     """Pickles the spec so that it can be unpickled on a subsequent run."""
+    flags, self.FLAGS = self.FLAGS, None
```

Comment:
Since the flags specified in the config are part of the config, those get correctly serialized. The command line flags don't get serialized, and have to be re-specified. This is the existing behavior though when using run stages.